NEWS

```diff
@@ -1,3 +1,84 @@
+Version 1.1.0 - July 2021
+
+- Add support for NVSHMEM communication for the Dslash operators, for
+  significantly improved strong scaling.  See
```

Comment:
```suggestion
  significantly improved strong scaling. See
```

Reply:
Ha, if you ask me to change every double space to single space after a period, you're asking for a global edit 😉 

This is an age thing it seems:
> The use of single vs. double-spacing after periods has become a great generational debate, with everyone from Gen X on up falling predominantly onto Team Double Space, while Millennials and Gen Z fight on the side of the single space.

https://www.thesaurus.com/e/writing/is-there-1-space-or-2-after-a-period/

Reply:
Great to know that... I never know you can put two spaces after a period.

Reply:
In other words: I am old and you are not 😜

Reply:
Haha ...

Reply:
Why waste the extra bytes on extra spaces? =)

Reply:
It makes me feel young :-)